app/src/main/java/org/fossasia/pslab/adapters/ControlMainAdapter.java

```diff
@@ -13,17 +17,37 @@
  * Created by asitava on 6/6/17.
  */
 
-public class ControlMainAdapter extends RecyclerView.Adapter<ControlMainAdapter.ViewHolder> {
+public class ControlMainAdapter extends RecyclerView.Adapter<ControlMainAdapter.ViewHolder> implements View.OnClickListener{
```

Comment:
The problem I was facing is I have three buttons with ```OnClickListener``` and only one ```OnClickListener``` for the buttons is actually working, whereas in case of edittext and seekbar, since are just one in number they work fine. So, i wrote it.